core/src/main/kotlin/org/evomaster/core/problem/rest/RestActionBuilderV3.kt

```diff
@@ -723,7 +723,8 @@ object RestActionBuilderV3 {
         }
     }
 
-    fun getBasePathFromURL(serverUrl: String): String {
+    fun getBasePathFromURL(swagger: OpenAPI): String {
+        val serverUrl = swagger.servers[0].url
```

Comment:
the comment:
`TODO would need more general approach, as different HTTP servers could
            have different base paths`
should be moved here